arm-network/2017-06-01/swagger/azureWAN.json

```diff
@@ -0,0 +1,2434 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "WANAsAServiceManagementClient",
+        "description": "REST API for Azure WAN As a Service.",
+        "version": "2017-06-01"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [
+        {
+            "azure_auth": [
+                "user_impersonation"
+            ]
+        }
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroup/{resourceGroupName}/providers/Microsoft.Network/WANs/{wanName}": {
+            "get": {
+                "operationId": "WANs_Get",
+                "x-ms-examples": {
+                    "WANGet": { "$ref": "../examples/WANGet.json" }
+                },
+                "description": "Retrieves the details of a WAN.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "name": "wanName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the WAN being retrieved."
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Request successful. Returns the details of the WAN retrieved.",
+                        "schema": {
+                            "$ref": "#/definitions/WAN"
```

Comment:
Yes, but since this model is used by NRP all over their APIs I didn't comment on making the change. We can work on getting this addressed later. Makes sense?